ons_alpha/topics/models.py

```diff
@@ -87,7 +88,28 @@ def latest_articles(self) -> QuerySet[ArticlePage]:
 
     @cached_property
     def latest_methodologies(self) -> QuerySet[MethodologyPage]:
-        return MethodologyPage.objects.live().public().child_of(self).order_by("-last_revised_date")[:5]
+        return MethodologyPage.objects.live().public().child_of(self).order_by("-last_revised_date", "-pk")[:5]
+
+    @cached_property
+    def related_by_topic(self) -> dict[str, QuerySet]:
+        related = {}
+        if bulletins := self.latest_by_series(BulletinPage, BulletinSeriesPage, self.topic):
+            related = {_("Bulletins"): bulletins}
+
+        if articles := self.latest_by_series(ArticlePage, ArticleSeriesPage, self.topic):
+            related = {_("Articles"): articles}
+
+        methodologies_qs = (
+            MethodologyPage.objects.live()
+            .public()
+            .not_child_of(self)
+            .filter(topics__topic=self.topic)
+            .order_by("-last_revised_date", "-pk")
+        )
+        if methodologies := methodologies_qs:
+            related = {_("Methodologies"): methodologies}
```

Comment:
`related` is being overridden every time, so only one of these is showing.
```suggestion
        if bulletins := self.latest_by_series(BulletinPage, BulletinSeriesPage, self.topic):
            related |= {_("Bulletins"): bulletins}

        if articles := self.latest_by_series(ArticlePage, ArticleSeriesPage, self.topic):
            related |= {_("Articles"): articles}

        methodologies_qs = (
            MethodologyPage.objects.live()
            .public()
            .not_child_of(self)
            .filter(topics__topic=self.topic)
            .order_by("-last_revised_date", "-pk")
        )
        if methodologies := methodologies_qs:
            related |= {_("Methodologies"): methodologies}
```

Reply:
argh, wrong choice in rebase. sorted